tensorflow-op-codegen/src/main.rs

```diff
@@ -270,6 +452,21 @@ fn define_op<W: Write>(
             c_name: attr.name.clone(),
         });
     }
+    for output in op.output_arg.iter() {
+        let mut number_attr = output.number_attr.clone();
+        let mut number_attr_opt = None;
+        if !number_attr.is_empty() {
+            number_attr_opt = Some(number_attr.clone());
+        }
+        op_outputs.push(Output {
+            //NOTE: we should be able to reuse attr keywords, correct me if im wrong.
```

Comment:
This should probably use a new `Escaper`.  Reusing `attr_escaper` means that if an op has an attr named `foo` and an output named `foo` then the generated code would be `BlahOutputs::foo_()` (i.e. with an underscore), even though they shouldn't actually clash.

Reply:
patching now, Ill also do this for Inputs in the next PR